src/Razor/src/Microsoft.VisualStudio.RazorExtension.Dependencies/AssemblyCodeBases.cs

```diff
@@ -7,115 +7,93 @@
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.LanguageServer.Protocol.Internal.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.LanguageServer.Protocol.Extensions.dll")]
 
-#if INCLUDE_ROSLYN_DEPS
 [assembly: ProvideBindingRedirection(
     AssemblyName = "Microsoft.CodeAnalysis",
     GenerateCodeBase = true,
     OldVersionLowerBound = "4.4.0.0",
-    OldVersionUpperBound = "4.6.0.0",
-    NewVersion = "4.6.0.0")]
```

Comment:
📝 These were unnecessarily complex. The tooling knows how to derive the values from the referenced assemblies at build time.